Releases.md

```diff
@@ -6,6 +6,40 @@ https://github.com/denoland/deno/releases
 We also have one-line install commands at
 https://github.com/denoland/deno_install
 
+### v0.18.0 / 2019.09.13
+
+In deno:
+
+- build: remove tools/build.py; cargo build is the build frontend now (#2865,
+  #2874, #2876)
+- feat: Added normalized benchmark graphs (#2919)
```

Comment:
This is not a feat that is part of the (any) release.

Reply:
removed